util/trunc/trunc.go

```diff
@@ -0,0 +1,9 @@
+package trunc
```

Comment:
Can you rename package to `text` or `strings` in case we will need to add more text related functions?

Reply:
✔️